galgebra/mv.py

```diff
@@ -2473,6 +2493,11 @@ def Fmt(self, fmt=1, title=None, dop_fmt=None):
                 return s
         return
 
+    def odot(self,dot_flg=True):
+        new_self = copy.deepcopy(self)
+        new_self.dot_flg = dot_flg
+        return new_self
```

Comment:
This is the only change to the differential operators part of this file (xref gh-133)